logprep/processor/domain_label_extractor/factory.py

```diff
@@ -15,15 +18,16 @@ def create(name: str, configuration: dict, logger) -> DomainLabelExtractor:
         """
         DomainLabelExtractorFactory._check_configuration(configuration)
 
-        domain_label_extractor = DomainLabelExtractor(name,
-                                                      configuration.get('tree_config'),
-                                                      configuration.get('tld_lists', None),
-                                                      configuration.get('tagging_field_name', 'tags'),
-                                                      logger)
-        domain_label_extractor.add_rules_from_directory(configuration['rules'])
-
-        return domain_label_extractor
+        return DomainLabelExtractor(
+            name=name,
+            configuration=configuration,
+            tld_lists=configuration.get("tld_lists", None),
```

Comment:
the `configuration` is part of the parameter list. there is no need to extract parts of it as seperate parameters. It is possibly the right time to refactor the `__init__` method of `DomainLabelExtractor` ;)